composer.json

```diff
@@ -117,6 +117,7 @@
         "bower-asset/highlight": "9.7.0",
         "embed/embed": "2.7.7",
         "npm-asset/jquery.caret": "^0.3.1",
-        "npm-asset/diff": "^3.4"
+        "npm-asset/diff": "^3.4",
+        "drupal/recaptcha": "2.2"
```

Comment:
Maybe we can move `drupal/recaptcha` more to the top where also the other drupal dependencies are

Reply:
Also we need to add this to `.make` file

Reply:
Done :-)